### PR TITLE
deleted comments fixed reference and merge conflict

### PR DESCRIPTION
**Pull request by es2055**

### CODE REVIEW

File: jobCentWeb/Frontend/src/scss/components/transfer.css [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+  padding-left: 15px;
+  width: 100%;
+  padding-top: 11px;
+  padding-right: 20px;
+}
+
+.transfer-input-field {
+  border-radius: 0;
+  border: none;
+  outline: #ccc solid 1px;
```
Comment (nakamolto) [inline jobCentWeb/Frontend/src/scss/components/transfer.css]: please clean up css

**Review by nakamolto — COMMENTED** [review]
please clean up css otherwise lgtm

File: jobCentWeb/Frontend/src/scss/components/transfer.css (hunk 1 above)
Comment (nucat176) [reply jobCentWeb/Frontend/src/scss/components/transfer.css]: still needs to be alphabetized

**Review by nucat176 — APPROVED** [review]
Looks good. Thanks Don. @mikebarile for approval

**Pull request merged**